.github/workflows/linux-tests.yml

```diff
@@ -38,7 +39,32 @@ jobs:
            channels: conda-forge,defaults
```

Comment:
Just in case, I think the error in the CI when installing Spyder from its repo (`git` option) comes from this config missing the channel/label where `spyder-kernels` betas are being added: `conda-forge/label/spyder_kernels_rc`